backend/app/assets/javascripts/spree/backend/turbolinks-configuration.js.erb

```diff
@@ -0,0 +1,20 @@
+// comment line necessary for correct interpolation of use_turbolinks
+Turbolinks.supported = <%= Spree::Backend::Config.use_turbolinks %>;
```

Comment:
Thanks for the feedback! I will sort out the rest if we get the #1900 merged in. 

On this one, we can copy over the [few lines from turbolinks](https://github.com/turbolinks/turbolinks/blob/c73e134731ad12b2ee987080f4c905aaacdebba1/src/turbolinks/index.coffee#L9-L12), but I really hate to encourage people to use old browsers, and those unsupported browsers are already ice age by now, let alone by the time somebody actually starts using this feature here..

So, well pointed out; my preference is to keep this behaviour.